e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -141,7 +150,7 @@ describe("scenarios > dashboards > filters > auto apply", () => {
   });
 
   it("should not preserve draft parameter values when editing the dashboard", () => {
-    createDashboard({ auto_apply_filters: false });
+    createDashboard({ dashboardDetails: { auto_apply_filters: false } });
```

Comment:
use a similar prop name in E2E test utils, the reason I nest this under `dashboardDetails` because I have introduced a new property `parameter`
https://github.com/metabase/metabase/blob/1a11947af5edb3a9ef9be7527997745218c3d828/e2e/support/commands/api/composite/createNativeQuestionAndDashboard.js#L3